src/pcrlock/pcrlock.c

```diff
@@ -1973,6 +1973,19 @@ static const char *ansi_true_color(uint8_t r, uint8_t g, uint8_t b, char ret[sta
         return ret;
 }
 
+/* red and green colors usually have good/bad meaning. So exclude color ranges that look too red- or
+ * greenish. See https://en.wikipedia.org/wiki/Hue */
+static double map_pcr_to_hue(uint32_t pcr) {
+        double exclude_red_range = 60.0;
```

Comment:
i think the red range could probably be smaller than the green range. to the eye more range appears than red (i.e. on red we quickly consider something orange or violet, while a lot is just green)